### DISCUSSION TITLE
createCookieSessionStorage in Cloudflare Pages

### DISCUSSION THREAD

**Discussion by cleytoncarvalho** [remix-run/remix#6160] [Q&A] [answered]
Is there any problem if I call createCookieSessionStorage in every request? The question is because I need to pass to it my SESSION_SECRET that is in the context that I receive from my loader/action.

I'm using Remix to deploy on Cloudflare Pages.

**Comment by sergiodxa** [comment, marked as answer]
There’s no issue, and even if you didn’t need to pass the secret inside the request handler in Serverless/Edge everything is instantiated on every request anyway.
